crates/net/network/benches/tx_manager_hash_fetching.rs

```diff
@@ -63,9 +60,7 @@ pub fn tx_fetch_bench(c: &mut Criterion) {
                             let peer_pool = peer.pool().unwrap();
 
                             for _ in 0..num_tx_per_peer {
-                                let mut gen = TransactionGenerator::new(
-                                    TestRng::deterministic_rng(RngAlgorithm::ChaCha),
-                                );
+                                let mut gen = TransactionGenerator::new(rand::rng());
```

Comment:
please use [`rand::rngs::StdRng`](https://docs.rs/rand/latest/rand/rngs/struct.StdRng.html) for deterministic rng